### PR TITLE
Docs: Mention `hasPartialFlowRev` and performance problem

### PR DESCRIPTION
**Pull request by RasmusWL**
The things that I mentioned in https://github.com/github/codeql/pull/6502#issuecomment-901087620 that never got into the document :flushed:

### CODE REVIEW

File: docs/codeql/writing-codeql-queries/debugging-data-flow-queries-using-partial-flow.rst [hunk 1]
```diff
@@ -113,4 +117,4 @@ Further reading
 ----------------
 
 - :ref:`About data flow analysis <about-data-flow-analysis>`
-- :ref:`Creating path queries <creating-path-queries>`
\ No newline at end of file
+- :ref:`Creating path queries <creating-path-queries>`
```
Comment (RasmusWL) [inline docs/codeql/writing-codeql-queries/debugging-data-flow-queries-using-partial-flow.rst:120]: I don't know why this line changed :flushed:

**Review by owen-mc — DISMISSED** [review]

File: docs/codeql/writing-codeql-queries/debugging-data-flow-queries-using-partial-flow.rst [hunk 2]
```diff
@@ -87,6 +89,8 @@ As noted in the documentation for ``hasPartialFlow`` (for example, in the
 
 This defines the exploration radius within which ``hasPartialFlow`` returns results.
 
+To get good performance when using ``hasPartialFlow`` it is important to ensure the ``isSink`` predicate of the configuration has no results. Likewise when using ``hasPartialFlowRev`` the ``isSource`` predicate of the configuration should have no results
```
Comment (jf205) [inline docs/codeql/writing-codeql-queries/debugging-data-flow-queries-using-partial-flow.rst:92]:
```suggestion
To get good performance when using ``hasPartialFlow`` it is important to ensure the ``isSink`` predicate of the configuration has no results. Likewise, when using ``hasPartialFlowRev`` the ``isSource`` predicate of the configuration should have no results.
```

**Review by jf205 — DISMISSED** [review]
Approved, pending one small suggestion. 👍🏻

File: docs/codeql/writing-codeql-queries/debugging-data-flow-queries-using-partial-flow.rst (hunk 2 above)
Comment (felicitymay) [inline docs/codeql/writing-codeql-queries/debugging-data-flow-queries-using-partial-flow.rst:92]: Missing period.

```suggestion
To get good performance when using ``hasPartialFlow`` it is important to ensure the ``isSink`` predicate of the configuration has no results. Likewise when using ``hasPartialFlowRev`` the ``isSource`` predicate of the configuration should have no results.
```

**Comment by RasmusWL** [comment]
Thanks for the reviews :+1: Had to re-request one after accepting the suggestion :shrug:

**Review by jf205 — APPROVED** [review]

**Pull request merged**